VaticanBAV/et/BAVet119.xml

```diff
@@ -45,16 +45,16 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                   Four circles, divided into four parts, fill the space between the arms.
                                   Future events are indicated with this figure.</title>
                           </msItem>
-                          <msItem xml:id="ms_i1.3" ref="LIT1171Awdana#tables">
+                          <msItem xml:id="ms_i1.3">
                               <locus from="4r" to="11v"/>
-                              <title>Sixteen circles, each divided into sixteen parts.</title>
+                              <title type="complete" ref="LIT1171Awdana#tables">Sixteen circles, each divided into sixteen parts.</title>
                               <incipit xml:lang="gez">
                                   ፩ ዓውደ፡ ሆይ። ሀ ፩ትረክቦ፡ ለዜና፡ መፍቅድ፡ በባሕረ፡ ፃና።
                               </incipit>
                           </msItem>
-                          <msItem xml:id="ms_i1.4" ref="LIT1171Awdana#seas">
+                          <msItem xml:id="ms_i1.4">
                               <locus from="13r" to="36v" />
-                              <title>Sixteen chapters, indicated with the names of local lakes:
+                              <title type="complete" ref="LIT1171Awdana#seas">Sixteen chapters, indicated with the names of local lakes:
                                   Ṣānā, Kʷarā, ʾElā, Šaklā, Zʷāy, ʾAlzazo, Wənǧ, Goǧǧam, Dāgo, 
```

Comment:
Should it be Goǧǧām? (Maybe not in this case.)

Reply:
It should be Goǧǧām, but the catalogue has Goǧǧam (p. 479-480).

Reply:
Ok, so maybe leave it as it is then (or add a `<sic resp="MV">`, but maybe that is exaggerated :) )

Reply:
Good idea, thank you Jonas